packages/firebase_crashlytics/CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
+## 0.1.1
```

Comment:
```suggestion
## 0.1.0+1
```

Justification: This made a bunch of improvements without changing the public API of the public.